packages/vuepress-theme-odyssey/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@okta/vuepress-theme-odyssey",
+  "version": "0.6.0",
+  "description": "VuePress theme for Odyssey Documentation",
+  "author": "Okta, Inc.",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint . --ext .js,.vue && stylelint ./*.scss",
+    "test": "yarn lint"
+  },
+  "devDependencies": {
+    "@okta/odyssey": "^0.6.0",
+    "@okta/odyssey-icons": "^0.6.0",
+    "@okta/prism-theme-odyssey": "^0.6.0",
+    "eslint-config-vuepress": "^2.2.0",
+    "html-loader": "^1.1.0",
+    "node-sass": "4.13.1",
+    "prismjs": "^1.14.0",
+    "sass-loader": "8.0.2",
+    "stylelint": "^9.3.0",
+    "stylelint-checkstyle-formatter": "^0.1.1",
+    "stylelint-config-recommended": "^2.1.0",
+    "stylelint-config-sass-guidelines": "^5.0.0",
+    "stylelint-config-standard": "^18.2.0",
+    "stylelint-order": "^0.8.1",
+    "stylelint-scss": "^3.2.0",
+    "vue-fragment": "^1.5.1",
+    "vuepress-plugin-clean-urls": "^1.1.1"
+  },
+  "dependencies": {
+    "focus-trap": "^6.1.3",
+    "focus-trap-vue": "https://github.com/arnoldsandoval-okta/focus-trap-vue.git#with-build"
```

Comment:
This fork of `focus-trap-vue` was necessary because I added functionality to the package. I submitted a pull request to the package repository which adds the functionality. It has been accepted and merged, I will update this when it is available via npm.

https://github.com/posva/focus-trap-vue/pull/302

Reply:
Ah - just left a comment about this. Glad you opened up a PR!